File: src/main/java/com/elissamc/api/MenuSystem/ChestInventory.java
package com.elissamc.api.MenuSystem;

import cn.nukkit.inventory.InventoryHolder;
import cn.nukkit.inventory.InventoryType;
import cn.nukkit.utils.TextFormat;

import java.util.HashMap;

public class ChestInventory extends FakeBaseInventory {
    private String owner;
    private String name;

    public ChestInventory(InventoryHolder holder, InventoryType type, String owner, String name, int size) {
        super(holder, type, new HashMap<>(), size);
        this.owner = owner;
        this.name = TextFormat.colorize(name);
    }

    @Override
    public String getName() {
        return name;
    }

    public void setName(String s){
        this.name = s;
    }

    public String getOwner() {
        return owner;
    }


}

File: src/main/java/com/elissamc/api/MenuSystem/ChestInventory.kt
package com.elissamc.api.MenuSystem

import cn.nukkit.inventory.InventoryHolder
import cn.nukkit.inventory.InventoryType
import cn.nukkit.item.Item
import cn.nukkit.utils.TextFormat

import java.util.HashMap

class ChestInventory(holder: InventoryHolder, type: InventoryType, val owner: String, n: String, size: Int) : FakeBaseInventory(holder, type, HashMap<Int, Item>(), size) {
    private var named: String = TextFormat.colorize(n)

    override fun getName(): String? {
        return named
    }

    fun setName(s: String) {
        this.named = s
    }


}
